pkgs/development/python-modules/clickhouse-cityhash/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "clickhouse-cityhash";
+  version = "1.0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0z8nl0ly2p1h6nygwxs6y40q8y424w40fkjv3jyf8vvcg4h7sdrg";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
+
+  doCheck = true;
```

```suggestion
  doCheck = false;
```

This package does not contain any tests.